sktime/forecasting/sarimax.py

```diff
@@ -6,6 +6,8 @@
 __all__ = ["SARIMAX"]
 __author__ = ["TNTran92"]
```

Comment:
you should add yourself as an author.
(this is not an "owned" estimator, so this is by contribution history)